packages/pictograms/pictograms.yml

```diff
@@ -4064,6 +4384,16 @@
     - life science
     - research
     - medicine
+- name: microservices
```

Comment:
This is named `microservices` here, but the SVG is named `microservices--02`. Which one should be updated?